docs/src/main/asciidoc/logging.adoc

```diff
@@ -176,18 +176,20 @@ opening the door to optimization opportunities.
 As an example, in native execution the minimum level enables lower level checks (e.g. `isTraceEnabled`) to be folded to `false`,
 resulting in dead code elimination for code that will never to be executed.
 
-All possible properties are listed in <<loggingConfigurationReference, the logging configuration reference>>.
+All possible properties are listed in the xref:loggingConfigurationReference[logging configuration reference] section.
 
-NOTE: If you are adding these properties via command line make sure `"` is escaped.
+NOTE: If you are adding these properties by using command-line, make sure `"` is escaped.
 For example `-Dquarkus.log.category.\"org.hibernate\".level=TRACE`.
 
 === Logging categories
 
-Logging is done on a per-category basis.  Each category can be independently configured.
+Logging is done on a per-category basis.
+Each category can be independently configured.
 A configuration which applies to a category will also apply to all sub-categories of that category,
 unless there is a more specific matching sub-category configuration.
 For every category the same settings that are configured on ( console / file / syslog ) apply.
-These can also be overridden by attaching a one or more named handlers to a category. See example in <<category-named-handlers-example>>
+These can also be overridden by attaching a one or more named handlers to a category.
+See example in the xref:category-named-handlers-example[] output.
```

Comment:
Should the label be...?

```suggestion
See example in the xref:category-named-handlers-example[Named handlers attached to a category] output.
```

Reply:
Why to write it there when it will be displayed by default and it renders fine?
![image](https://user-images.githubusercontent.com/48474054/222497970-fc332d15-83bf-4425-aaed-db1314960822.png)

Reply:
@MichalMaler - That's what I thought... but other updates here do have labels so wanted to ask to be sure. Thanks :+1: